### ISSUE TITLE
Faust2alqt execution error report

### ISSUE THREAD

**Issue by simon28li** [grame-cncm/faust#726]
hi，When I execute `faust2alqt -qrcode examples/analysis/ vumeter.dsp`, the error is as follows,Can you give me some support? thank you very much.

```
vumeter.cpp: In function ‘int main(int, char**)’:
vumeter.cpp:20069:9: error: ‘cerr’ was not declared in this scope
         cerr << "Unable to init audio" << endl;
         ^~~~
vumeter.cpp:20069:9: note: suggested alternative:
In file included from vumeter.cpp:50:0:
/usr/include/c++/7/iostream:62:18: note:   ‘std::cerr’
   extern ostream cerr;  /// Linked to standard error (unbuffered)
                  ^~~~
vumeter.cpp:20069:43: warning: ‘QTextStream& QTextStreamFunctions::endl(QTextStream&)’ is deprecated: Use Qt::endl [-Wdeprecated-declarations]
         cerr << "Unable to init audio" << endl;
                                           ^~~~
In file included from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/qdebug.h:49:0,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/qcborcommon.h:45,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/qcborvalue.h:45,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/qcborarray.h:43,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/QtCore:38,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtGui/QtGuiDepends:3,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtGui/QtGui:3,
                 from vumeter.cpp:4177:
/home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/qtextstream.h:293:75: note: declared here
 Q_CORE_EXPORT QT_DEPRECATED_VERSION_X(5, 15, "Use Qt::endl") QTextStream &endl(QTextStream &s);
                                                                           ^~~~
vumeter.cpp:20076:9: error: ‘cerr’ was not declared in this scope
         cerr << "Unable to start audio" << endl;
         ^~~~
vumeter.cpp:20076:9: note: suggested alternative:
In file included from vumeter.cpp:50:0:
/usr/include/c++/7/iostream:62:18: note:   ‘std::cerr’
   extern ostream cerr;  /// Linked to standard error (unbuffered)
                  ^~~~
vumeter.cpp:20076:44: warning: ‘QTextStream& QTextStreamFunctions::endl(QTextStream&)’ is deprecated: Use Qt::endl [-Wdeprecated-declarations]
         cerr << "Unable to start audio" << endl;
                                            ^~~~
In file included from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/qdebug.h:49:0,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/qcborcommon.h:45,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/qcborvalue.h:45,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/qcborarray.h:43,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/QtCore:38,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtGui/QtGuiDepends:3,
                 from /home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtGui/QtGui:3,
                 from vumeter.cpp:4177:
/home/spack/opt/spack/linux-sles15-aarch64/gcc-7.4.0/qt-5.15.2-7rrn4ijudadgahilepwh5pfggrjlbj7u/include/QtCore/qtextstream.h:293:75: note: declared here
 Q_CORE_EXPORT QT_DEPRECATED_VERSION_X(5, 15, "Use Qt::endl") QTextStream &endl(QTextStream &s);
                                                                           ^~~~
make: *** [Makefile:580: vumeter.o] Error 1

```

**Comment by sletz** [comment]
Probably a couple of typos, should be fixed in https://github.com/grame-cncm/faust/commit/cf80f4c94ac979922892a7f594121358cd178708. Can you test and report?

**Comment by simon28li** [comment]
> Probably a couple of typos, should be fixed in [cf80f4c](https://github.com/grame-cncm/faust/commit/cf80f4c94ac979922892a7f594121358cd178708). Can you test and report?

@sletz Great. Problem solved. Thanks for your hard work.
